### PR TITLE
add Orange and OSGi pointers in the readme.md

### PR DESCRIPTION
**Pull request by Nicolas-Lingois**

### CODE REVIEW

**Comment by maggu2810** [comment]
It seems your commit misses a correct signed-off-by line.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Eclipse SmartHome now hosts the Orange implementation of a standard OSGi EnOcean base driver.
```
Comment (kaikreuzer) [inline README.md]: I would reformulate it to:
"This repository hosts the reference implementation for the OSGi EnOcean Device Service specification, which is a sub-project of [Eclipse SmartHome](https://www.eclipse.org/smarthome/) that has been originally implemented and contributed by Orange."

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+Eclipse SmartHome now hosts the Orange implementation of a standard OSGi EnOcean base driver.
+
+Orange implementation of OSGi standard EnOcean Device service specification is now available and maintained on eclipse SmartHome git repository under eclipse public licence (https://www.eclipse.org/legal/epl-v10.html). The specification is officially a chapter of OSGi Release 6 Residential specification (http://www.osgi.org/Release6/HomePage). The implementation of an OSGi Device Service specification is called an OSGi base driver. A base driver simplifies the development of applications controlling devices of a specific protocol, here the EnOcean standard radio technology (https://www.enocean-alliance.org).
```
Comment (kaikreuzer) [inline README.md]: I would remove the first sentence here as it is a repetition from the sentence above now.

File: README.md (hunk 1 above)
Comment (maggu2810) [inline README.md]: That could be confusing (at least I think so), as the "that has been ..." could be point to the ESH as well.

"This repository hosts the reference implementation for the OSGi EnOcean Device Service specification, that has been originally implemented and contributed by Orange and is a sub-project of Eclipse SmartHome ."

Comment (kaikreuzer) [inline README.md]: Ok :-)

Comment (Nicolas-Lingois) [inline README.md]: Ok so I just remove the first sentence and we're good ^^

**Comment by kaikreuzer** [comment]
Thanks - there is no valid CLA for nicolas.lingois@mythalesgroup.com - could you please check?

**Comment by Nicolas-Lingois** [comment]
oh I see ... I registered my eclipse account with my personal email ... I changed the email address with my professional one ! Should i resend a pull request or you can validate as is ?

**Comment by kaikreuzer** [comment]
No worries, I can manually check, what I just did and I can confirm that the CLA is now available. Thanks!

**Pull request merged**